htdocs/compta/facture/contact.php

```diff
@@ -52,7 +52,8 @@
 $object = new Facture($db);
 // Load object
 if ($id > 0 || !empty($ref)) {
-	$ret = $object->fetch($id, $ref, '', '', $conf->global->INVOICE_USE_SITUATION);
+		$object->fetch($id);
```

Comment:
Why removing the other parameters likee $ref ?

Reply:
indeed after reading the doc; i probably need to readd them (i added those line by copy / pasting some code from another script not using them.

So let's readd them. So i propably shoud remove the fetch_thirdparty just bellow.

Reply:
fixed thx for your review eldy